src/egg/core/eggColorFader.cpp

```diff
@@ -0,0 +1,140 @@
+#include <egg/core/eggColorFader.h>
+
+#include <sdk/gx/GXAttr.h>
```

Comment:
If including all these headers bother you consider what OGWS does: https://github.com/kiwi515/ogws/blob/master/include/revolution/GX.h